tests/sles4sap/publiccloud/mr_test_setup_env.pm

```diff
@@ -0,0 +1,60 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Set up public cloud instance testing environment for 'mr_test'
+# Maintainer: QE-SAP <qe-sap@suse.de>
+# Tags: TEAM-6709
+
+use warnings;
+use Mojo::Base 'publiccloud::basetest';
+use publiccloud::ssh_interactive qw(select_host_console prepare_ssh_tunnel);
+use testapi;
+use Mojo::File 'path';
+use publiccloud::utils;
+use Data::Dumper;
+
+sub test_flags {
+    return {
+        fatal => 1,
+        milestone => 0,
+        publiccloud_multi_module => 1
+    };
+}
+
+sub run {
+    my ($self, $run_args) = @_;
+    my $mr_test_tar = 'mr_test-master.tar.gz';
+    my $instance = $run_args->{my_instance};
+
+    # Select console on the host, not the PC instance
+    select_host_console();
+
+    # Preserve args for post_fail_hook
+    $self->{provider} = $run_args->{my_provider};    # Required for cleanup
```

Comment:
you don't have to do this , @mimi1vx  found a better way , see https://github.com/os-autoinst/os-autoinst-distri-opensuse/commit/02f132014f2d87725d5bf2f29bb40b56764da3a8

Reply:
Deleted. Thanks for the info :+1: